examples/source/1.components/amp-web-push/static/sw.js

```diff
@@ -18,17 +18,9 @@ importScripts('/sw-toolbox/sw-toolbox.js');
 importScripts('https://cdn.rawgit.com/jakearchibald/idb/97e4e878/lib/idb.js');
```

Comment:
Was linting/fixing this file on purpose?

Reply:
yes - is there a reason why it shouldn't be?

Reply:
Not really 😉was thinking about browser compatibility but every browser that does support SWs should also support modern syntax.